.circleci/config.yml

```diff
@@ -18,4 +38,13 @@ jobs:
           command: eval $(opam config env) && opam repo add remote-default https://opam.ocaml.org && opam pin add -y obeam obeam && opam pin add -y fialyzer .
       - run:
           name: Test
-          command: eval $(opam config env) && make test
+          # NOTE: Don't run `make test` because `make test` requires Erlang/OTP.
+          command: eval $(opam config env) && make unit-test && make blackbox-test
+workflows:
```

Comment:
docs: 

- https://circleci.com/blog/persisting-data-in-workflows-when-to-use-caching-artifacts-and-workspaces/
- https://circleci.com/docs/2.0/workflows/
- https://circleci.com/docs/2.0/configuration-reference/#workflows